types/vue.d.ts

```diff
@@ -111,8 +111,8 @@ export interface VueConstructor<V extends Vue = Vue> {
   component<Props>(id: string, definition: FunctionalComponentOptions<Props, RecordPropsDefinition<Props>>): ExtendedVue<V, {}, {}, {}, Props>;
   component(id: string, definition?: ComponentOptions<V>): ExtendedVue<V, {}, {}, {}, {}>;
 
-  use<T>(plugin: PluginObject<T> | PluginFunction<T>, options?: T): VueConstructor<V>;
-  use(plugin: PluginObject<any> | PluginFunction<any>, ...options: any[]): VueConstructor<V>;
+  use<T>(plugin: PluginObject<T> | PluginFunction<T>, ...options: T[]): VueConstructor<V>;
+  use(plugin: PluginObject<unknown> | PluginFunction<unknown>, ...options: unknown[]): VueConstructor<V>;
```

Comment:
Could you add a comment on top of this stating that this makes options mandatory when they are defined

Reply:
That is done I guess